### PR TITLE
Reformatted code.

### PR DESCRIPTION
**Pull request by ChamaraPhilipsuom**
As requested by code reviews.

### CODE REVIEW

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit-local.jsp [hunk 1]
```diff
@@ -71,11 +71,11 @@
...
-                    .Authenticator.SAML2SSO.DESTINATION_URL_PREFIX);
-            if (destinationURLList.size() == 0){
+            destinationURLList = IdPManagementUIUtil.getPropertySetStartsWith(properties,
+                    IdentityApplicationConstants.Authenticator.SAML2SSO.DESTINATION_URL_PREFIX);
+            if (destinationURLList.size() == 0) {
                 destinationURLList.add(IdPManagementUIUtil.getProperty(properties,
-                    IdentityApplicationConstants.Authenticator.SAML2SSO.SSO_URL));
+                        IdentityApplicationConstants.Authenticator.SAML2SSO.SSO_URL));
             }
         } else if(IdentityApplicationConstants.OAuth10A.NAME.equals(federatedAuthenticator.getName())){
```
Comment (omindu) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit-local.jsp]: Formatting issue.

Comment (ChamaraPhilipsuom) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit-local.jsp]: Fixed.

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/domain.jsp [hunk 2]
```diff
@@ -26,7 +26,7 @@
         if (request.getParameter("authFailureMsg") != null) {
             errorMessage = request.getParameter("authFailureMsg");
 
-            if (errorMessage.equalsIgnoreCase("domain.unknown")) {
+            if ("domain.unknown".equalsIgnoreCase(errorMessage)) {
```
Comment (Jasintha) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/domain.jsp]: better to void of using string middle of the code as much as you can , also IMO this error message should be support for localization isn't it ?

Comment (ChamaraPhilipsuom) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/domain.jsp]: Fixed at #1682.

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/domain.jsp [hunk 3]
```diff
@@ -16,7 +16,11 @@
   ~ under the License.
   --%>
 <%@ page import="org.owasp.encoder.Encode" %>
+<%@ page import="java.util.ResourceBundle" %>
 <%
+    String BUNDLE = "org.wso2.carbon.identity.application.authentication.endpoint.i18n.Resources";
+    ResourceBundle resourceBundle = ResourceBundle.getBundle(BUNDLE, request.getLocale());
+    String domain_unknown = resourceBundle.getString("domain.unknown");
```
Comment (Kishanthan) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/domain.jsp]: we don't normally use these type of naming conventions. It goes normally like "domainUnknown"

Comment (ChamaraPhilipsuom) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/domain.jsp]: Fixed at #1682

**Pull request merged**